zendesk/ticket_test.go

```diff
@@ -163,3 +164,32 @@ func TestCreateTicket(t *testing.T) {
 		t.Fatalf("Returned ticket does not have the expected ID %d. Ticket id is %d", expectedID, ticket.ID)
 	}
 }
+
+func TestUpdateTicket(t *testing.T) {
+	mockAPI := newMockAPIWithStatus(http.MethodPut, "ticket.json", http.StatusOK)
+	client := newTestClient(mockAPI)
+	defer mockAPI.Close()
+
+	trg, err := client.UpdateTicket(ctx, 2, Ticket{})
+	if err != nil {
+		t.Fatalf("Failed to get ticket: %s", err)
+	}
+
+	expectedID := int64(2)
+	if trg.ID != expectedID {
+		t.Fatalf("Returned ticket does not have the expected ID %d. Ticket id is %d", expectedID, trg.ID)
+	}
+}
+
+func TestUpdateTicketFailure(t *testing.T) {
+	mockAPI := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
you could probably just use this function here...

https://github.com/nukosuke/go-zendesk/blob/master/zendesk/zendesk_test.go#L40-L45

Could just pick one of the fixtures to return and force the 500 with the status parameter

Reply:
Removed tests serving no real function